packages/components-angular/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts

```diff
@@ -319,10 +340,22 @@ export class SwissPostIntranetHeaderComponent implements OnInit, OnChanges, Afte
     }
   }
 
-  public getPlaceholderSearchIntranet() {
+  public get searchInputPlaceholder() {
     return this.localization['searchPlaceholder'][this.lang.toLowerCase()];
   }
 
+  public get searchButtonLabel() {
+    return this.localization['searchButton'][this.lang.toLowerCase()];
+  }
+
+  public get showNavigationLabel() {
+    return this.localization['showNavigationLabel'][this.lang.toLowerCase()];
+  }
+
+  public get hideNavigationLabel() {
+    return this.localization['hideNavigationLabel'][this.lang.toLowerCase()];
+  }
+
   public isLanguageActive(lang: string) {
     return this.lang.toLowerCase() === lang.toLowerCase();
```

Comment:
It could be a good idea, to create a `this.langLow = this.lang.toLowerCase()` variable in the constructor, to reduce function calls every single time we need the lang lowercased.
What do you think?

Reply:
I don't see the changes I requested, but anyway...
As long as everything is working I'm fine with it.